### DISCUSSION TITLE
Can not spawn language server on Windows

### DISCUSSION THREAD

**Discussion by supengchuan** [mrcjkb/rustaceanvim#168] [Q&A] [answered]
Hello, 

I use lazy.nvim as pugin manager, install rustaceanvim by
```lua
{
  'mrcjkb/rustaceanvim',
  version = '^3', -- Recommended
  ft = { 'rust' },
}
```

when I open a rust project, the error occurs:

![image](https://github.com/mrcjkb/rustaceanvim/assets/18416902/eed59167-54f7-42f3-9488-d7d1fa693abb)

I have already installed rust-analyzer. 
![image](https://github.com/mrcjkb/rustaceanvim/assets/18416902/124f8ca2-41f3-469a-bd9d-a3084e5b2c3e)

My OS is windows 11,  and  use the default config of rustaceanvim. 

Thanks for your replay

**Comment by mrcjkb** [comment]
Hey :wave:

You can check whether rustaceanvim can find rust-analyzer by running `:checkhealth rustaceanvim`.

If it can't, you either need to use a different way to install rust-analyzer, or to add it to the `PATH` so that Neovim can find it.

**Reply by supengchuan** [reply, marked as answer]
Thank you for your reply, after some testing, I found that it may be caused by the rust-analyzer I installed using mason, and I may not be very clear about the specific problem. I manually downloaded rust-analyzer from github and put it under $PATH and it was ready to go
